### DISCUSSION TITLE
Fork Another Branch

### DISCUSSION THREAD

**Discussion by DarshanSP19** [community/community#55898] [New to GitHub]
I have forked one public repository and It had clone only main branch of the upstream repository. Now I want to pull another branch of upstream repository and It's not letting me create another fork as I already have one fork present in my Account. 

What to do?

**Comment by airtower-luna** [comment]
The easiest thing is to work with a local repository:

* Clone the repository (or use an existing copy you already have)
* Fetch from the upstream repository (so you have current versions of all upstream branches)
* Push the right branch to your fork
